src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -992,7 +992,7 @@ private function registerRequestConfiguration(array $config, ContainerBuilder $c
 
     private function registerAssetsConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader)
```

Comment:
Here we now need to change the type-hint for `$loader` to `PhpFileLoader` and ensure that in in `load()` we pass the `$phpLoader`.